testplan/cli/commands/writers.py

```diff
@@ -16,6 +17,7 @@
 
 
 writer_commands = CommandList()
+export_context = ExportContext()
```

Comment:
module level export context might not be the best idea, as it putting all the exporters into a single context, I'm not sure that is what we always want. I suggest to keep separate unique contexts for now for tpr.